options.c

```diff
@@ -1267,6 +1284,44 @@ static void set_format(void *data, const char *buf)
 	update_full();
 }
 
+static void print_re_error(int errcode, regex_t *re, const char *re_src) {
+	size_t len;
+	char *buf;
```

Comment:
IMO it is better to not declare variables separately; this looks like very old C.